package.json

```diff
@@ -466,6 +472,21 @@
           "when": "view == flowr-dependencies",
           "group": "navigation@3"
         },
+        {
+          "command": "vscode-flowr.feedback",
+          "when": "view == flowr-dependencies",
+          "group": "navigation@3"
+        },
+        {
+          "command": "vscode-flowr.session.connect",
+          "when": "view == flowr-dependencies && !isWeb",
+          "group": "navigation@3"
+        },
+        {
+          "command": "vscode-flowr.session.disconnect",
+          "when": "view == flowr-dependencies && !isWeb",
+          "group": "navigation@3"
+        },
```

Comment:
i like the idea but i thin these buttons reduce the clarity of the header row - i thin we should not add them